demo/src/screens/select/multiselect.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
couple strange things happening here:

1. if you try to select an option that is above a checkbox, the option is not selected but the checkbox is checked: http://recordit.co/9Mk9KlrMJu

2. if chocolate is already selected and it is then disabled, it remains selected: http://recordit.co/Ex7tScCjAx. presumably it would be removed.

if this is not meant to be a reusable component its not that big of a deal but these issues may pop up elsewhere

Reply:
I fixed the issue selecting an option above the checkbox. The issue was the z-index of the invisible input inside the checkbox so just set wrapper z-index to 0.